scripts/net-traffic

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -Eeu -o pipefail
```

Comment:
Interesting!  What's going on here?

Reply:
`-E` makes sure we're inheriting errors from subshells
`-e` exits the script immediately upon an error (i.e. it doesn't "bleed" its errors either into userspace or, worse, the bar)
`-u` treats unset variables as errors (i.e. you have to initialize them or the script won't run)
`-o pipefail` fails the script with the status of the last pipeline execution (and not of the overall script)

Reply:
Generally, these are safe to use when you're running a properly written script